uniffi-bindgen-react-native.podspec

```diff
@@ -9,7 +9,11 @@ Pod::Spec.new do |s|
   s.license          = { :type => package['license'], :file => 'LICENSE' }
   s.author           = { package['author']['name'] => package['author']['email'] }
   s.source           = { :git => package['repository']['url'], :tag => s.version.to_s }
-  s.platform     = :ios, '9.0'
-  s.source_files  = 'cpp/includes/*.{h,cpp,hpp}'
-  s.dependency 'React'
+  s.platform         = :ios, '13.0'
+  s.source_files     = 'cpp/includes/*.{h,cpp,hpp}'
+  s.swift_versions   = '4.0'
+  s.pod_target_xcconfig = {
+    'SWIFT_OPTIMIZATION_LEVEL' => '-Onone',
```

Comment:
This struck me as slightly unexpected. Why do we disable compiler optimization here?

Reply:
We don't use any Swift, and it was recommended by `pod lib lint`. I think if/when we start using Swift we should revisit this optimization.